src/loaders/svg/tvgXmlParser.cpp

```diff
@@ -144,10 +207,11 @@ bool simpleXmlParseAttributes(const char* buf, unsigned bufLength, simpleXMLAttr
     if (!buf || !func) return false;
 
     while (itr < itrEnd) {
-        const char* p = _simpleXmlSkipWhiteSpace(itr, itrEnd);
+        const char* p = itr;
```

Comment:
How about changing only function?
p = _skipWhiteSpacesAndXmlEntities(itr, itrEnd);
Are there any cases where this would cause problems?

Reply:
it's a remnant of an earlier version of the changes :)
thx